src/App.jsx

```diff
@@ -1,107 +1,32 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function findUserByPosts(userId) {
+  return usersFromServer.find(user => (
+    user.id === userId
+  )) || null;
+}
+
+function getCommentsByPosts(postId) {
+  return commentsFromServer.filter(comment => (
+    comment.postId === postId
+  ));
+}
+
+export const posts = postsFromServer.map(post => ({
```

Comment:
Would be better const posts give name more readable , 
for example preparedPosts.